### DISCUSSION TITLE
A1111 extension for comfyui!

### DISCUSSION THREAD

**Discussion by PladsElsker** [comfyanonymous/ComfyUI#216] [Show and tell]
We made an extension for A1111 to embed comfyui in a tab. Please check it out! 

https://github.com/ModelSurge/sd-webui-comfyui

![ezgif com-video-to-gif(1)](https://user-images.githubusercontent.com/34081873/226529347-23e61102-cf83-457e-b94c-89337fd38c4d.gif)

**Comment by GrimblyGorn** [comment]
Ooh. I was wondering if something like this was possible. Seems it was :)

Gonna check it out later. Thanks :)

**Reply by tufeixp** [reply]
Can it support call webui extensions like : https://github.com/thygate/stable-diffusion-webui-depthmap-script.git  ?

**Comment by Uwnodi** [comment]
Does not work with.  https://github.com/lshqqytiger/stable-diffusion-webui-directml
